keras_metrics/metrics.py

```diff
@@ -65,8 +69,8 @@ class true_positive(layer):
     positive class.
     """
 
-    def __init__(self, name="true_positive", **kwargs):
-        super(true_positive, self).__init__(name=name, **kwargs)
+    def __init__(self, label=None, sparse=False, name="true_positive", **kwargs):
```

Comment:
I believe that it would be better to put the ```sparse``` and ```label``` parameters after the ```name```, so the following code will work both for the current and for the future version:
```py
import keras_metrics as km
tp = true_positive("true_positive_metric")
```

Otherwise, people get a breaking change after the library update.